src/Coordination/KeeperServer.cpp

```diff
@@ -173,6 +174,12 @@ struct KeeperServer::KeeperRaftServer : public nuraft::raft_server
         reconfigure(new_config);
     }
 
+    void commit_in_bg() override
+    {
+        MemoryTrackerBlockerInThread blocker;
```

Comment:
Need a comment.

Reply:
I think this should be `LockMemoryExceptionInThread`. We want to prevent exception from being thrown and not ignore memory allocations.

Reply:
Yeah, should be OK: https://github.com/ClickHouse/ClickHouse/blob/f608e62c27bd5c830b87d646758081def912224f/src/IO/WriteBuffer.h#L122